hyperemitter.js

```diff
@@ -26,7 +26,7 @@ function HyperEmitter (db, schema, opts) {
     return new HyperEmitter(db, schema, opts)
   }
 
-  this.messages = protobuf(schema)
+  this.codecs = protobuf(schema)
```

Comment:
let's just stop accepting a schema and wrapping protobuf, we will just support codecs, and otherwise we emit buffers.